src/collections/Breadcrumb/BreadcrumbDivider.js

```diff
@@ -18,7 +18,7 @@ function BreadcrumbDivider(props) {
   const classes = cx(className, 'divider')
   const ElementType = getElementType(BreadcrumbDivider, props)
 
-  if (icon) return createIcon(icon, { ...rest, className: classes })
+  if (icon) return Icon.create(icon, { ...rest, className: classes })
```

Comment:
Example static shorthand create method for the Icon.  I think this cohesion makes more sense anyhow.  Keeping the `mapValueToProps` method in the same file as the component that it is creating.